docs/resources/container.md

```diff
@@ -122,7 +122,7 @@ The `device` block supports:
 * `name` - *Required* - Name of the device.
 
 * `type` - *Required* - Type of the device Must be one of none, disk, nic,
-	unix-char, unix-block, usb, gpu, infiniband, proxy.
+	unix-char, unix-block, usb, gpu, infiniband, proxy, unix-hotplug, tpm.
```

Comment:
Why is pci missing here?

Reply:
My mistake, I was under the impression that the container was restricted to being a container and wasn't also allowed to be a vm.